### ISSUE TITLE
Think about backend architecture and UI/UX interactions

### ISSUE THREAD

**Issue by frankPairs** [Fellowship-5/Idiomia#7]
In order to start the project, it's always nice to have an overview of the solution that you are proposing. It does not have to be very detailed because you can define these details before starting a specific task.

In terms of API, you can write in a paper about:

- Which kind of models you have in your business.
- What data you want to save in the database.
- How these models are related to each other.
- Important constraints in your business (for example, can only an admin approve a proverb?).

Diagrams can help you to accomplish this.

In terms of UI/UX, you can write in a paper about:

- Mockup about navigation between pages.
- What information you want to show on each page.
- Use a  UI design library or build the UI components yourselves.

Just move this task to "Done" column when you feel ready.

Good luck!
